cparser/Parser.vy

```diff
@@ -114,10 +114,10 @@ primary_expression:
 | var = VAR_NAME
     { (VARIABLE (fst var), snd var) }
 | cst = CONSTANT
-    { (CONSTANT (fst cst), snd cst) }
+    { (Cabs.CONSTANT (fst cst), snd cst) }
```

Comment:
I'm a bit sad that some Cabs constructors  are qualified with `Cabs.` (because they must be) but others are not.  This could make this grammar slightly harder to read than if all constructors were consistently prefixed.